tensorflow/core/kernels/fake_quant_ops.cc

```diff
@@ -43,6 +46,9 @@ using tensorflow::errors::InvalidArgument;
 namespace tensorflow {
 
 typedef Eigen::ThreadPoolDevice CPUDevice;
+#ifdef TENSORFLOW_USE_SYCL
+typedef Eigen::SyclDevice SYCLDevice;
```

Comment:
Can be moved to the ifdef line 35

Reply:
We usually try to keep the symmetry with the other devices so I think it's better not to move it. Also namespace is different.